contents/product-engineers/your-first-ab-test.md

```diff
@@ -0,0 +1,86 @@
+---
+title: How to think about your first A/B test (why, when, and where)
+date: 2024-01-11
+author: ["ian-vanagas"]
+featuredImage: ../images/blog/experiment-hog.jpg
+featuredImageType: full
+tags: 
+  - AB testing
+  - Product engineers
+---
+
+Most A/B testing products won't tell you this, but everyone doesn't need to be A/B testing all of the time. Because of this, it can be tricky to figure out why, when, and where you should run one, especially if you haven't run one before. To help you make these decisions, we've put together this guide on running your first A/B test.
+
+> Want background about A/B testing and how to run a good one? Check out [our A/B testing guide for engineers](/product-engineers/ab-testing-guide-for-engineers). We skip how A/B tests work and their specific implementation in this guide.
+
+## Why run your first A/B test?
+
+The first question you must answer when thinking about A/B testing is **why?** An obvious answer is that you want to improve your product and business, but there are many other ways to do this. A/B tests are technically free, but they have hidden costs like:
+
+- The work it takes to get up to speed with A/B testing concepts and systems.
+- The time to plan, implement, and coordinate A/B tests.
+- The potential distraction A/B tests are on higher impact, strategic work.
+
+If these costs don't scare you away, A/B tests are a great way to measure the impact of non-strategic changes such as positioning, design, and flows (signup, subscription, onboarding). They aren't going to make or break your product or business, but they make it a lot better if they succeed. Specific benefits for these types of changes include:
+
+- **Significant improvements to your most important metrics.** Ideally, A/B tests aim to improve and optimize your metrics like signups, conversion, subscriptions, and revenue. Every successful test directly makes your company more successful.
+
+- **Clarity on the impact of your changes.** A/B tests provide precise stats on a change's impact on your goal metrics. Rather than looking at a range of feedback and metrics to understand the impact of a change, you can look at one test.
+
+- **Protection from the downside.** Changes always come with risks. They can negatively impact all the same metrics you are trying to improve. By tracking a specific change's impact, ****A/B tests help prevent you from shipping changes that make your product worse.
```

Comment:
```suggestion
1. **Significant improvements to your most important metrics.** Ideally, A/B tests aim to improve and optimize your metrics like signups, conversion, subscriptions, and revenue. Every successful test directly makes your company more successful.

2. **Clarity on the impact of your changes.** A/B tests provide precise stats on a change's impact on your goal metrics. Rather than looking at a range of feedback and metrics to understand the impact of a change, you can look at one test.

3. **Protection from the downside.** Changes always come with risks. They can negatively impact all the same metrics you are trying to improve. By tracking a specific change's impact, ****A/B tests help prevent you from shipping changes that make your product worse.
```